ddp-workspace/projects/ddp-dsm-ui/src/app/participant-page/participant-page.component.html

```diff
@@ -1131,13 +1132,33 @@ <h3 class="Color--warn Line--Break">Lost to Follow-Up</h3>
                     <hr style="border-color: black">
                     <fieldset style="border: 1px black solid">
                       <b class="Group--Title">{{setting.columnDisplay}}</b>
+                      <ng-container *ngIf="checkBoxGroups[setting.columnName] !== null">
+                        <table class="Width--100">
+                          <tr>
+                            <td class="TD--Padding Width--30">
+                            </td>
+                            <td>
+                              <ng-container *ngFor="let field of checkBoxGroups[setting.columnName]; let fieldIndex = index">
+                                <app-form-data [fieldSetting]="field" [participantData]="getParticipantForDynamicField(field)"
+                                               [activityData]="getActivityData(field)"
+                                               [activityOptions]="getActivityOptions(field)"
+                                               [checkBoxGroups]="checkBoxGroups"
+                                               (patchData)="formPatch($event, field, setting, findDataId(setting))"></app-form-data>
+                              </ng-container>
+                            </td>
+                          </tr>
+                        </table>
+                      </ng-container>
                       <ng-container *ngFor="let field of settings[setting.columnName]; let fieldIndex = index">
-                        <app-form-data [fieldSetting]="field" [participantData]="getParticipantForDynamicField(field)"
-                                       [activityData]="getActivityData(field)"
-                                       [activityOptions]="getActivityOptions(field)"
-                                       (patchData)="formPatch($event, field, setting, findDataId(setting))"></app-form-data>
-                        <hr *ngIf="fieldIndex !== settings[setting.columnName].length-1"
-                            style="border-color: lightgray">
+                        <ng-container *ngIf="field.displayType !== 'CHECKBOX'">
+                          <app-form-data [fieldSetting]="field" [participantData]="getParticipantForDynamicField(field)"
+                                         [activityData]="getActivityData(field)"
+                                         [activityOptions]="getActivityOptions(field)"
+                                         [checkBoxGroups]="checkBoxGroups"
+                                         (patchData)="formPatch($event, field, setting, findDataId(setting))"></app-form-data>
+                          <hr *ngIf="fieldIndex !== settings[setting.columnName].length-1"
```

Comment:
If you try to avoid the style for the last item in the list it can be simplified a bit, like:
```
 <ng-container *ngFor="let field of settings[setting.columnName]; let last = last">
   <hr *ngIf="!last" >
   ...
```

`fieldIndex ` is not needed as well here.